app/code/Magento/Quote/Model/Quote/Item/AbstractItem.php

```diff
@@ -599,6 +603,26 @@ public function setConvertedPrice($value)
         return $this;
     }
 
+    /**
+     * @param bool $status
+     *
+     * @return AbstractItem
+     */
+    public function setHasChildren(bool $status): AbstractItem
+    {
+        $this->setData('has-children', $status);
```

Comment:
Such change is backward-incompatible, by the way...

I see `$this->setHasChildren(true);` in code, there is no need in explicit methods implementation, just add them as `@method` annotation.